templates/security/login.html.twig

```diff
@@ -26,7 +26,7 @@
 
     <input type="hidden" name="_csrf_token" value="{{ csrf_token('authenticate') }}">
 
-    <div class="checkbox mb-3"></div><input type="checkbox" name="_remember_me"> {{ 'RememberMe'|trans }}</label>
+    <div class="checkbox mb-3"><label><input type="checkbox" name="_remember_me"> {{ 'RememberMe'|trans }}</label>
```

Comment:
Mist de closing `div` nu niet?

Reply:
Kwam deze in een latere PR ook tegen, is opgelost

Reply:
Aleen mis je dan nu weer de <label>  open tag...